### PR TITLE
Try to fix MAM flaky test

### PR DESCRIPTION
**Pull request by marcofavorito**
## Proposed changes

Try to fix MAM flaky test, e.g.:

https://github.com/fetchai/agents-aea/pull/2556/checks?check_run_id=2824321866#step:8:41817
https://github.com/fetchai/agents-aea/pull/2556/checks?check_run_id=2824321671#step:8:62118

## Fixes

n/a

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes and CI passes too
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

n/a

### CODE REVIEW

File: tests/test_manager/test_manager.py [hunk 1]
```diff
@@ -566,27 +566,12 @@ class TestMultiAgentManagerAsyncMode(
     """Tests for MultiAgentManager in async mode."""
 
 
-class TestMultiAgentManagerAsyncModeWithPassword(
-    BaseTestMultiAgentManager
-):  # pylint: disable=unused-argument,protected-access,attribute-defined-outside-init
-    """Tests for MultiAgentManager in async mode, with password."""
-
-    PASSWORD = "password"  # nosec
```
Comment (DavidMinarsch) [inline tests/test_manager/test_manager.py:574]: Why removed?

File: tests/test_manager/test_manager.py [hunk 2]
```diff
@@ -566,27 +566,12 @@ class TestMultiAgentManagerAsyncMode(
...
     """Tests for MultiAgentManager in threaded mode."""
 
     MODE = "threaded"
 
 
-class TestMultiAgentManagerThreadedModeWithPassword(BaseTestMultiAgentManager):
-    """Tests for MultiAgentManager in threaded mode, with password."""
-
-    MODE = "threaded"
-    PASSWORD = "password"  # nosec
```
Comment (DavidMinarsch) [inline tests/test_manager/test_manager.py:587]: Why removed?

File: tests/test_manager/test_manager.py (hunk 1 above)
Comment (marcofavorito) [reply tests/test_manager/test_manager.py:574]: trying to debug the problem on CI. The problem still occurs though.

**Comment by marcofavorito** [comment]
The problem isn't easy to debug, as locally all the tests pass.

**Comment by DavidMinarsch** [comment]
Closing in favour of #2571

**Pull request closed without merging**